docs/dev/libs/cep.md

```diff
@@ -334,6 +334,20 @@ pattern.where(event => ... /* some condition */).or(event => ... /* or condition
 </div>
 </div>
 
+
+**Stop condition:** In case of kleene operators (`oneOrMore()` and `oneOrMore().optional()`) you can
+also specify a stop condition, e.g. accept events with value larger than 5 until the sum of values is smaller than 50.
```

Comment:
In general, I would not use the "kleene" term, as we do not use it throughout the rest of the docs.
You can say sth like "looping" or simply oneOrMore.
In addition, it would also be more consistent to not use "operator" but "pattern". In the docs we have "pattern sequences" composed of "patterns" so we should stick to that.

So "Kleene Operator" -> "looping pattern" or "oneOrMore pattern"